cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/Filters/FilterCollection/FilterConditionInput/index.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { StyleRules, withStyles, WithStyles } from '@material-ui/core/styles';
+import { CustomOperator } from 'components/ConfigurationGroup/types';
+import Heading, { HeadingTypes } from 'components/Heading';
+import If from 'components/If';
+import { OPERATOR_VALUES } from 'components/PluginJSONCreator/constants';
+import PluginInput from 'components/PluginJSONCreator/Create/Content/PluginInput';
+import { ICreateContext, IWidgetInfo } from 'components/PluginJSONCreator/CreateContextConnect';
+import * as React from 'react';
+
+const styles = (): StyleRules => {
+  return {
+    filterConditionInput: {
+      '& > *': {
```

Comment:
We should avoid this.. If this component will have child component this basically affects all. This defeats the purpose of using css-in-js.

Reply:
Resolved

Reply:
My bad! I made this change in later PRs but I must have missed it in this PR.